biojava-structure/src/main/java/org/biojava/nbio/structure/StructureImpl.java

```diff
@@ -249,7 +249,7 @@ public Chain findChain(String chainId, int modelnr) throws StructureException {
 
 		// iterate over all chains.
 		for (Chain c : chains) {
-			if (c.getChainID().equals(chainId)) {
+			if (c.getName().equals(chainId)) {
```

Comment:
Can you rename the argument to authId? I tried to make this consistent wherever I found this.